test/release/examples/benchmarks/shootout/fasta.chpl

```diff
@@ -96,47 +101,93 @@ proc repeatMake(desc, alu, n) {
 }
 
 //
-// Output a random sequence of length 'n' using distribution a
+// Output a random sequence of length 'n' using distribution 'a'
```

Comment:
Yeesh, no idea.  Maybe it used to be called 'a' and I improved it?  Where's my semantic comment checker?